openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -203,7 +203,7 @@ This is an example of a Wallet Instance Attestation:
    * MUST support protocol extensions for SD-JWT VC credential format profile as defined in this specification (#vc_sd_jwt_profile).
    * As a way to invoke the Wallet, at least a custom URL scheme `haip://` MUST be supported. Implementations MAY support other ways to invoke the wallets as agreed by trust frameworks/ecosystems/jurisdictions, not limited to using other custom URL schemes.
    * Response type MUST be `vp_token`.
-   * Response mode MUST be `direct_post`. The Verifier MUST return `redirect_uri` in response to the HTTP POST request from the Wallet, where the Wallet redirects the User to, as defined in Section 6.2 of [@!OIDF.OID4VP]. Implementation considerations for the response mode `direct_post` are given in Section 11.5 of [@!OIDF.OID4VP].
+   * Response mode MUST be `direct_post.jwt`. The Verifier MUST return `redirect_uri` in response to the HTTP POST request from the Wallet, where the Wallet redirects the User to, as defined in Section 6.2 of [@!OIDF.OID4VP]. Implementation considerations for the response mode `direct_post.jwt` are given in Section 12.4 of [@!OIDF.OID4VP].
```

Comment:
not honestly sure what it was supposed to be but 6.2 is now [6.2](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#section-6.2).  [Credential Set Query](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-credential-set-query)

Reply:
[7.2](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#section-7.2).  [Response Mode "direct_post"](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-response-mode-direct_post) ?

Reply:
[7.3.1](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#section-7.3.1).  [Response Mode "direct_post.jwt"](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html#name-response-mode-direct_postjw) ?

Reply:
I think it wants to reference the place where "redirect_uri" is defined, so I think 7.2 is right:

```suggestion
   * Response mode MUST be `direct_post.jwt`. The Verifier MUST return `redirect_uri` in response to the HTTP POST request from the Wallet, where the Wallet redirects the User to, as defined in Section 7.2 of [@!OIDF.OID4VP]. Implementation considerations for the response mode `direct_post.jwt` are given in Section 12.4 of [@!OIDF.OID4VP].
```